.github/workflows/pre-merge.yaml

```diff
@@ -1,43 +1,116 @@
-name: pre-merge check
+name: Pre Merge Check
 
 on:
+  push:
+    branches:
+      - OLH-2228
```

Comment:
will this need to be removed before merge?

Reply:
Thanks @saralk , leaving as is for now as it doesn't impact anything and part of the required code when we workflow block the pipeline, we would just change the branch to the main branch.